apps/heureka/src/components/Services/utils.ts

```diff
@@ -158,7 +169,7 @@ type NormalizedServiceImageVersions = {
 export const getNormalizedImageVersionsData = (
   data: GetServiceImageVersionsQuery | undefined
 ): NormalizedServiceImageVersions => ({
-  totalCount: data?.ComponentVersions?.totalCount || 0,
+  total: data?.ComponentVersions?.totalCount || 0,
```

Comment:
as there are so many `total`s this `total` may not convey the full meaning, I'd suggest to give it more context like `totalNumberOfImageVersions` or just `totalImageVersions` to not to confuse it with others.